app/models/forms/delete_logs_form.rb

```diff
@@ -0,0 +1,18 @@
+module Forms
+  class DeleteLogsForm
+    include ActiveModel::Model
+    include ActiveModel::Validations
+
+    attr_reader :logs, :log_type
+
+    def initialize(logs: nil, log_ids: nil, log_type: nil)
+      @logs = logs
+      @log_ids = log_ids
+      @log_type = log_type || (logs.first&.lettings? ? :lettings : :sales)
+    end
+
+    def no_logs_selected?
+      @log_ids.blank? || @log_ids.reject(&:blank?).blank?
+    end
```

Comment:
We should add validations here and instead use `valid?` externally to determine whether we're save to progress the deletion.